dotnet-testing/MockingIOptions/MockingIOptionsTests/CustomerServiceTests.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.Extensions.Options;
+using MockingIOptions.Configuration;
+using MockingIOptions.Services;
+using Moq;
+
+namespace MockingIOptionsTests
+{
+    public class CustomerServiceTests
+    {
+        private DatabaseConfiguration _configData;
+
+        [SetUp]
+        public void Setup()
+        {
+            _configData = new DatabaseConfiguration { ConnectionString = "TestConnectionString" };
```

Comment:
We can make `"TestConnectionString"` a constant on the class and then reuse it in each test where we are asserting this value as the expected result.